### ISSUE TITLE
10.10.3 with XCode 6.3 Command Line Tools?

### ISSUE THREAD

**Issue by barts** [bundler/bundler#3554]
Hi, I upgraded to OS 10.10.3 today and XCode 6.3 with the XCode 6.3 Command Line Tools from here:
https://developer.apple.com/downloads/index.action#

When I try to do a `bundle install` I get this error, which is so long GitHub won't let me paste it into this form (and the plain text file for it is 1.1MB large...):
https://www.dropbox.com/s/qotscp9lmkd8r0z/bundler.txt?dl=0

**Comment by indirect** [comment]
It looks like that error doesn't actually include the full text. Perhaps your terminal is running out of backscroll? Can you try running `bundle install > output.txt` and then upload the output.txt file to dropbox or a gist?

**Comment by TimMoore** [comment]
This gist is decoded from the URL parameter in the file linked above: https://gist.github.com/TimMoore/0ed2626e9dba2ae30639

It looks like a problem building the `debugger` gem. @barts does `gem install debugger` work for you?

**Comment by TimMoore** [comment]
Also @barts the `debugger` gem is not compatible with Ruby 2.0+. You might want to try `byebug` instead. See https://github.com/cldwalker/debugger#known-issues

**Comment by barts** [comment]
Installing `byebug` (and `pry-byebug` just in case) seems to have fixed it. Thank you!

**Comment by TimMoore** [comment]
I can't seem to reproduce whatever caused the excessively long error message. I get much more useful output when I try to build a `Gemfile` with the `debugger` gem on a newer Ruby:

```
Fetching gem metadata from https://rubygems.org/....
Fetching version metadata from https://rubygems.org/..
Resolving dependencies...
Installing columnize 0.9.0
Installing debugger-linecache 1.2.0
Installing debugger-ruby_core_source 1.3.8

Gem::Ext::BuildError: ERROR: Failed to build gem native extension.

    /Users/tmoore/.rubies/ruby-2.2.0/bin/ruby -r ./siteconf20150412-15238-1tt0hsp.rb extconf.rb
*** extconf.rb failed ***
Could not create Makefile due to some reason, probably lack of necessary
libraries and/or headers.  Check the mkmf.log file for more details.  You may
need configuration options.

Provided configuration options:
	--with-opt-dir
	--with-opt-include
	--without-opt-include=${opt-dir}/include
	--with-opt-lib
	--without-opt-lib=${opt-dir}/lib
	--with-make-prog
	--without-make-prog
	--srcdir=.
	--curdir
	--ruby=/Users/tmoore/.rubies/ruby-2.2.0/bin/$(RUBY_BASE_NAME)
/Users/tmoore/.rubies/ruby-2.2.0/lib/ruby/2.2.0/fileutils.rb:1391:in `initialize': No such file or directory @ rb_sysopen - ./220/ruby_debug.h (Errno::ENOENT)
	from /Users/tmoore/.rubies/ruby-2.2.0/lib/ruby/2.2.0/fileutils.rb:1391:in `open'
	from /Users/tmoore/.rubies/ruby-2.2.0/lib/ruby/2.2.0/fileutils.rb:1391:in `copy_file'
	from /Users/tmoore/.rubies/ruby-2.2.0/lib/ruby/2.2.0/fileutils.rb:485:in `copy_file'
	from /Users/tmoore/.rubies/ruby-2.2.0/lib/ruby/2.2.0/fileutils.rb:402:in `block in cp'
	from /Users/tmoore/.rubies/ruby-2.2.0/lib/ruby/2.2.0/fileutils.rb:1570:in `block in fu_each_src_dest'
	from /Users/tmoore/.rubies/ruby-2.2.0/lib/ruby/2.2.0/fileutils.rb:1584:in `fu_each_src_dest0'
	from /Users/tmoore/.rubies/ruby-2.2.0/lib/ruby/2.2.0/fileutils.rb:1568:in `fu_each_src_dest'
	from /Users/tmoore/.rubies/ruby-2.2.0/lib/ruby/2.2.0/fileutils.rb:401:in `cp'
	from extconf.rb:83:in `block in <main>'
	from extconf.rb:82:in `each'
	from extconf.rb:82:in `<main>'

extconf failed, exit code 1

Gem files will remain installed in /Users/tmoore/Projects/bundler/issues/issue-3554/.gem/ruby/2.2.0/gems/debugger-1.6.6 for inspection.
Results logged to /Users/tmoore/Projects/bundler/issues/issue-3554/.gem/ruby/2.2.0/extensions/x86_64-darwin-14/2.2.0-static/debugger-1.6.6/gem_make.out
An error occurred while installing debugger (1.6.6), and Bundler cannot continue.
Make sure that `gem install debugger -v '1.6.6'` succeeds before bundling.
```

I have 10.10.3 and the latest command line tools, so that must not be be the only cause.

Anyway, since things are working for you now, I'll close this issue.

**Comment by sodabrew** [comment]
The official fix is Xcode 6.3.1 and/or Command Line Tools 6.3 with release date of Apr 21, 2015. The original buggy CLT 6.3 was released on Apr 9, 2015.
